cookbook/agent_concepts/knowledge/vector_db/lance_db/lance_db_hybrid_search.py

```diff
@@ -0,0 +1,32 @@
+from agno.agent import Agent
+from agno.knowledge.knowledge import Knowledge
+from agno.models.openai import OpenAIChat
+from agno.vectordb.lancedb import LanceDb, SearchType
+
+knowledge = Knowledge(
+    name="My PG Vector Knowledge Base",
+    description="This is a knowledge base that uses a PG Vector DB",
+    vector_store=LanceDb(
+        table_name="vectors",
+        uri="tmp/lancedb",
+        search_type=SearchType.hybrid,
+    ),
+)
+
+knowledge.add_content(
+    name="Recipes",
+    url="https://agno-public.s3.amazonaws.com/recipes/ThaiRecipes.pdf",
+    metadata={"user_tag": "Recipes from website"},
+)
+
+agent = Agent(
+    model=OpenAIChat(id="gpt-4o"),
+    knowledge=knowledge,
+    search_knowledge=True,
+    read_chat_history=True,
```

Comment:
Why though?

Reply:
i copied the pgvector cookbook and changed to lancedb

Reply:
removed